cypress/e2e/full/cashierLegacy/paymentAgentDepositScreenLoad.cy.js

```diff
@@ -18,9 +18,7 @@ screenSizes.forEach((screenSize) => {
     beforeEach(() => {
       cy.clearAllSessionStorage()
       cy.c_login({ user: 'cashierLegacyPA' })
-      cy.c_visitResponsive('appstore/traders-hub', screenSize, {
-        skipPassKeys: true,
-      })
+      cy.c_visitResponsive('appstore/traders-hub', screenSize, {})
```

Comment:
No need to pass empty obj here since it's there in default args.
```suggestion
      cy.c_visitResponsive('appstore/traders-hub', screenSize)
```